pkg/controllers/sidecar/config/config.go

```diff
@@ -79,15 +79,15 @@ func (s *ScyllaConfig) Setup(ctx context.Context) (*exec.Cmd, error) {
 // - cluster_name
 // - rpc_address
 // - endpoint_snitch
-func (s *ScyllaConfig) setupScyllaYAML() error {
+func (s *ScyllaConfig) setupScyllaYAML(scyllaYamlPath, scyllaYamlConfigMapPath string) error {
 	// Read default scylla.yaml
-	scyllaYAMLBytes, err := ioutil.ReadFile(scyllaYAMLPath)
+	scyllaYAMLBytes, err := ioutil.ReadFile(scyllaYamlPath)
```

Comment:
Do we really need to have a scyllaYAMLPath and a scyllaYamlPath? For once, YAML is the common way to write acronyms in go. Second, it is not clear what the difference between the two are.

If we just need one in this function, it is fine to name parameter scyllaYAMLPath and let it hide the outer one. If we need both, we need a better name.

Reply:
renamed variables.